usecases/Concrete/simulation_model/material_problems/linear_elasticity.py

```diff
@@ -0,0 +1,108 @@
+import dolfin as df
+
+from usecases.Concrete.simulation_model.material_problems.template_material import MaterialProblem
```

Comment:
Just for the naming, what is a Material Problem? a structural FEM-model?

Reply:
basically, yes.